### PR TITLE
[DE-566] geo_s2 analyzer test case

### PR DESCRIPTION
**Pull request by aMahanna**
DE-566 seeks to introduce `geo_s2` analyzer creation support.

Given that the `database.create_analyzer` method accepts `analyzer_type` as a parameter, there is no feature implementation required.

Nevertheless a test case has been added to ensure `python-arango`'s support for `geo_s2` analyzer creation (on EE >=3.10.5 only)

### CODE REVIEW

File: tests/test_analyzer.py [hunk 1]
```diff
@@ -56,3 +58,11 @@ def test_analyzer_management(db, bad_db, cluster):
 
     # Test delete missing analyzer with ignore_missing set to True
     assert db.delete_analyzer(analyzer_name, ignore_missing=True) is False
+
+    # Test create geo_s2 analyzer (EE only)
+    if enterprise and db_version >= version.parse("3.10.5"):
+        analyzer_name = generate_analyzer_name()
+        result = db.create_analyzer(analyzer_name, "geo_s2", {})
+        assert result["type"] == "geo_s2"
+        assert result["properties"]["format"] == "latLngDouble"
```
Comment (alexbakharew) [inline tests/test_analyzer.py:67]: Could you please check other default values for this analyzer? Default options are the following:
```
{ 
  "options" : { 
    "maxCells" : 20, 
    "minLevel" : 4, 
    "maxLevel" : 23 
  }, 
  "type" : "shape", 
  "format" : "latLngDouble" 
}
```

**Review by alexbakharew — CHANGES_REQUESTED** [review]
Please check all analyzer parameters

File: tests/test_analyzer.py (hunk 1 above)
Comment (aMahanna) [reply tests/test_analyzer.py]: 👍 added as of https://github.com/ArangoDB-Community/python-arango/pull/263/commits/6e2b00a461dc0072517c4e1c9d0a934da8763d85

**Review by alexbakharew — APPROVED** [review]
LGTM

**Pull request merged**